careers/application-process/index.html

```diff
@@ -0,0 +1,292 @@
+                                <span class="icon-link_text">See the history of the CFPB</span>
+                            </a>
+                        </footer>
+                    </div>
+                </article>
+            </div>
+        </div>
+        {% endcall %}
+    </section>
+{%- endblock %}
+
+{% block content_main %}
+    <section class="block block__flush-top block__sub">
+            <h1>Job Application Process</h1>
+            <p class="h3">
+                Applying for a federal job can take some time and patience.
+                It is a process that prioritizes both fairness to all applicants
+                and getting the best qualified candidates.
+                Watch the video above (or read the transcript)
+                for an overview of how it works.
+            </p>
+            <p class="short-desc">
+                Please note that to ensure this process is followed,
+                we can’t take applications directly.
+                You must submit your application in one of the ways
+                described on USAJobs.
+                Here are some things to keep in mind:
+            </p>
+    </section>
+
+    <section class="job-applications-interfaces">
+        <h2>Different interfaces you may have to use</h2>
+        <div class="expandable
+                    expandable__expanded
+                    expandable__mobile-only
+                    u-mb0">
+            <button class="expandable_header
+                           expandable_header__jump-link
+                           expandable_target
+                           u-show-on-mobile">
+                <span class="expandable_header-left expandable_label">
+                    Current Openings
+                </span>
+                <span class="expandable_header-right expandable_link">
+                    <span class="expandable_cue-open">
+                        <span class="cf-icon cf-icon-plus-round"></span>
+                    </span>
+                    <span class="expandable_cue-close">
+                        <span class="cf-icon cf-icon-minus-round"></span>
+                    </span>
+                </span>
+            </button>
+            <div class="expandable_content">
+                <div class="media block block__flush-bottom">
+                    <div class="media_image-container media_image-container__wide-margin">
+                        <img class="media_image u-centered-on-mobile"
+                             src="/static/img/computer2-round.png">
+                    </div>
+                    <div class="media_body">
+                        <h3 class="u-hide-on-mobile">Current Openings</h3>
+                        <p class="short-desc">
+                            We post new job announcements on our website.
+                            You can always check back here for more.
+                        </p>
```

Comment:
From @schaferjh: Since this section is about describing the different interfaces and not sending them there, there's no need for the link. ~However~ the copy might need to be changed to make that more clear. It'll get brought up in user testing.